app/src/main/java/com/babylon/wallet/android/presentation/settings/authorizeddapps/dappdetail/DappDetailScreen.kt

```diff
@@ -494,28 +495,28 @@ fun DAppWebsiteAddressRow(
             style = RadixTheme.typography.body1Regular,
             color = RadixTheme.colors.gray2
         )
-        websiteAddresses.forEach { address ->
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .clickable {
-                        context.openUrl(address)
-                    },
-                verticalAlignment = Alignment.CenterVertically,
-                horizontalArrangement = Arrangement.spacedBy(dimensions.paddingSmall)
-            ) {
-                Text(
-                    modifier = Modifier.weight(1f),
-                    text = address,
-                    style = RadixTheme.typography.body1HighImportance,
-                    color = RadixTheme.colors.blue1
-                )
-                Icon(
-                    painter = painterResource(id = com.babylon.wallet.android.designsystem.R.drawable.ic_link_out),
-                    tint = RadixTheme.colors.gray3,
-                    contentDescription = null
-                )
-            }
+        Row(
```

Comment:
we are no longer displaying list of addresses here?

Reply:
Matt mentioned that in the previous week's Wednesday standup. As per metadata standards we will show one website which we need to validate against the well-known file. iOS is doing the same.